ui/tests/integration/components/ttl-picker-test.js

```diff
@@ -1,30 +1,248 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render, fillIn } from '@ember/test-helpers';
-import hbs from 'htmlbars-inline-precompile';
+import { render, click, fillIn } from '@ember/test-helpers';
 import sinon from 'sinon';
+import hbs from 'htmlbars-inline-precompile';
 
-module('Integration | Component | ttl picker', function (hooks) {
+module('Integration | Component | ttl-picker', function (hooks) {
```

Comment:
Contents of the test from `ttl-picker2` replaced this test for a deleted component